accepted/00075-live-patching-enhancements.md

```diff
@@ -0,0 +1,170 @@
+
+#### Live patching for a system
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a SLES client from a combobox
+ 3. Select a kernel version from a dropdown (current kernel version of the system is preselected)
+
+The resulting filters will set up the project for live patching from the specified kernel version.
+If the wizard is accessed from the project view, all the created filters will be automatically added to the project.
+
+#### Live patching for a product
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a synced SLES product from a dropdown
+ 3. Select a kernel version available in the selected product (latest is preselected)
+
+The resulting filters will set up the project for live patching from the specified kernel version.
+If the wizard is accessed from the project view, all the created filters will be automatically added to the project.
+
+#### Additional enhancements
+
+##### CLM filter list
+
+CLM project wizards is aimed to make the project creation easier by automatically adding multiple filters. As a consequence, the list of filters in Uyuni can grow very quickly. To handle this problem, the following improvements can be made in the CLM filter list page:
+
+ - Add selection checkboxes to delete multiple filters at once
+ - Add "Delete unused/Select unused" shortcuts to delete or select all the filters that are not currently attached to any project
+
+##### Project wizards for other purposes
+
+The concept of CLM project wizards can be easily extended to other use cases as well. An example of such a use case is adding multiple AppStream module filters at once. A wizard called "AppStream modules with defaults" can be implemented with the following workflow:
+
+ 1. Input a label prefix for the resulting filters. If the wizard is accessed from the project view, prefill this field with the project label
+ 2. Select a modular channel from a dropdown
+ 3. In additional textboxes, add multiple module/stream pairs to override the default streams as required
+
+When done, the wizard creates a module filter per module in the repository, specifying the default stream for that module, or the specified stream if overridden in the wizard.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+**TODO**
+
+# Alternatives
+[alternatives]: #alternatives
+
+Alternative implementations for EOL data retrieval are mentioned in the [data retrieval](#data-retrieval) section.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+ - Which approach shall be used for [EOL data retrieval](#data-retrieval)
+ - What is the roadmap for `zypper lifecycle` and the lifecycle data format
+ - What is the best way to define a live patching project with the existing filters (see [Live patching CLM workflow](#live-patching-clm-workflow))
```

Comment:
- Do we need a "Project Wizard" or a "Filter Wizard" ?

Reply:
Filter wizard :mage_man: :+1: